autoload/test/php/dusk.vim

```diff
@@ -0,0 +1,43 @@
+if !exists('g:test#php#dusk#file_pattern')
+  let g:test#php#dusk#file_pattern = '\v(t|T)est.php$'
```

Comment:
Even though it probably wouldn't matter in the real world, it would be correct to escape the dot (`.`) here and write `\v(t|T)est\.php$`, as otherwise it means "any character".

Reply:
Good catch—I was wrestling with a folder-centric regex of this PR before deciding to just move the test runner to run before PHPUnit's—must've missed this in the shuffle.

I'll take care of these two reviews—would you prefer an amended commit, or a separate one that you can squash when merging?

Reply:
I'm good with an amended commit.